var/spack/repos/builtin/packages/py-blessed/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PyBlessed(PythonPackage):
+    """Blessed is a thin, practical wrapper around terminal capabilities in
+    Python."""
+
+    homepage = "https://github.com/jquast/blessed"
+    url      = "https://pypi.io/packages/source/b/blessed/blessed-1.15.0.tar.gz"
+
+    version('1.15.0', sha256='777b0b6b5ce51f3832e498c22bc6a093b6b5f99148c7cbf866d26e2dec51ef21')
+
```

Comment:
Missing dependency:
```python
depends_on('py-setuptools', type='build')
```

Reply:
py-blessed installs just fine without setup tools. Also, I tested it to be working via running the simple example indicated on github README. The spec for the installed package is show below. If you insist, I can just add the dependency. Thanks.
```
Concretized
--------------------------------
py-blessed@1.15.0%gcc@8.2.0 arch=linux-linuxmint19-x86_64
    ^py-six@1.12.0%gcc@8.2.0 arch=linux-linuxmint19-x86_64
        ^python@3.7.2%gcc@8.2.0+bz2+ctypes+dbm+lzma~nis~optimizations+pic+pyexpat+pythoncmd+readline+shared+sqlite3+ssl~tkinter~ucs4~uuid+zlib arch=linux-linuxmint19-x86_64
            ^bzip2@1.0.6%gcc@8.2.0+shared arch=linux-linuxmint19-x86_64
            ^expat@2.2.5%gcc@8.2.0+libbsd arch=linux-linuxmint19-x86_64
                ^libbsd@0.9.1%gcc@8.2.0 arch=linux-linuxmint19-x86_64
            ^gdbm@1.18.1%gcc@8.2.0 arch=linux-linuxmint19-x86_64
                ^readline@7.0%gcc@8.2.0 arch=linux-linuxmint19-x86_64
                    ^ncurses@6.1%gcc@8.2.0~symlinks~termlib arch=linux-linuxmint19-x86_64
            ^libffi@3.2.1%gcc@8.2.0 arch=linux-linuxmint19-x86_64
            ^openssl@1.0.2o%gcc@8.2.0+systemcerts arch=linux-linuxmint19-x86_64
                ^zlib@1.2.11%gcc@8.2.0+optimize+pic+shared arch=linux-linuxmint19-x86_64
            ^sqlite@3.28.0%gcc@8.2.0+fts~functions arch=linux-linuxmint19-x86_64
            ^xz@5.2.4%gcc@8.2.0 arch=linux-linuxmint19-x86_64
    ^py-wcwidth@0.1.7%gcc@8.2.0 arch=linux-linuxmint19-x86_64
```